assets/CSS/shopping-cart.css

```diff
@@ -1 +1,80 @@
-@import 'colors.css';
\ No newline at end of file
+@import "colors.css";
+a {
+  text-decoration: none;
+  color: white;
+}
+.btn-apply1 {
+  color: var(--secondary-color);
+  font-weight: 500;
+  background-color: white;
+  border: solid 2px var(--secondary-color);
+  transition: background-color 0.3s ease;
+}
+
+.btn-apply1:hover {
+  color: #4498e6;
```

Comment:
please use the color schema we have, change the color and background color to --secondary-color